### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by Bombergames**
adds comments object and shows them in a listview in odorevent details

### CODE REVIEW

File: android/app/src/main/java/edu/gatech/cs/environmentalodors/MapsActivity.java [hunk 1]
```diff
@@ -261,6 +263,13 @@ public void generateFakeData() {
             event.addOdorReport(tempOdorReport);
             EventBus.getDefault().post(new OdorReportEvent(tempOdorReport));
         }
+
+        for(int i = 0; i < commentCount; i++)
+        {
+            Comment tempComment = new Comment();
```
Comment (estebanlau) [inline android/app/src/main/java/edu/gatech/cs/environmentalodors/MapsActivity.java:269]: No line breaks before the opening braces. Change everywhere else as well.

File: android/app/src/main/java/edu/gatech/cs/environmentalodors/models/Comment.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ */
+
+public class Comment {
+    public String author;
+    public commentType type;
+    public String content;
+
+    public Comment(String author, String content, commentType type)
+    {
+        this.author = author;
```
Comment (estebanlau) [inline android/app/src/main/java/edu/gatech/cs/environmentalodors/models/Comment.java]: No line breaks before opening braces.

File: android/app/src/main/java/edu/gatech/cs/environmentalodors/models/OdorEvent.java [hunk 3]
```diff
@@ -30,9 +32,12 @@ public OdorEvent() {
         // We're allowed to create an odor event without an odor report for debugging purposes.
     }
 
+    public void addComment(Comment comment) {comments.add(comment);}
+
     public List<OdorReport> getOdorReports() {
         return odorReports;
     }
+    public List<Comment> getComments() { return comments; }
```
Comment (estebanlau) [inline android/app/src/main/java/edu/gatech/cs/environmentalodors/models/OdorEvent.java:42]: We should have consistent formatting. Insert a line break after opening braces.

File: android/app/src/main/java/edu/gatech/cs/environmentalodors/MapsActivity.java [hunk 4]
```diff
@@ -247,6 +248,7 @@ public void generateFakeData() {
         LatLng center = DEFAULT_LOCATION; // approximately atlanta
         float radius = 0.2f;
         int reportCount = 5;
+        int commentCount = 5;
```
Comment (estebanlau) [inline android/app/src/main/java/edu/gatech/cs/environmentalodors/MapsActivity.java:251]: Better practice to declare variables as close as possible to their location of use.

**Review by estebanlau — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/edu/gatech/cs/environmentalodors/MapsActivity.java (hunk 4 above)
Comment (Bombergames) [reply android/app/src/main/java/edu/gatech/cs/environmentalodors/MapsActivity.java:251]: I think it'll be fine this way - it allows easy editing of the test without having to sift through the code.  That is probably a good policy for non-tests though.

**Comment by Bombergames** [comment]
New commit adds a way to make comments throught ui, also cleaned up formatting and fixed adapter outdated data bug.

**Review by estebanlau — APPROVED** [review]

**Pull request merged**